org.qbittorrent.qBittorrent.yaml

```diff
@@ -20,6 +20,7 @@ finish-args:
   - --talk-name=org.freedesktop.Notifications
   - --talk-name=org.freedesktop.PowerManagement
   - --talk-name=org.gnome.SessionManager
+  - --own-name=org.kde.StatusNotifierItem-2-2
```

Comment:
The problem with this is the pid is the same in every sandbox so if another app is running they both collide.

Realistically this needs to patched somewhere (in Qt I presume) for this to reliably function.